src/containers/my-resources/attachments-container/AttachmentsContainer.tsx

```diff
@@ -44,17 +46,62 @@ const AttachmentsContainer = () => {
     [setAlert]
   )
 
+  const onDeleteAttachmentError = (error: ErrorResponse) => {
+    setAlert({
+      severity: snackbarVariants.error,
+      message: error ? `errors.${error.code}` : ''
+    })
+  }
+
+  const onDeleteAttachmentResponse = () => {
+    setAlert({
+      severity: snackbarVariants.success,
+      message: 'myResourcesPage.attachments.successDeletion'
+    })
+  }
+
   const getAttachments = useCallback(
     () => ResourceService.getAttachments({ limit: itemsPerPage }),
     [itemsPerPage]
   )
 
-  const { response, loading } = useAxios<ItemsWithCount<Attachment>>({
-    service: getAttachments,
-    defaultResponse: defaultResponses.itemsWithCount,
-    onResponseError: onAttachmentError
+  const deleteAttachment = useCallback(
+    (id?: string) => ResourceService.deleteAttachment(id ?? ''),
+    []
+  )
+
+  const { response, loading, fetchData } = useAxios<ItemsWithCount<Attachment>>(
```

Comment:
Add an alias for fetchData, the same as you added in fetchDeleteAttachment

Reply:
> Add an alias for fetchData, the same as you added in fetchDeleteAttachment

fixed